### ISSUE TITLE
[Bug]: Recovery phrase reveal not accessible via keyboard navigation

### ISSUE THREAD

**Issue by georgewrmarshall** [MetaMask/metamask-extension#33785]
### Describe the bug

The recovery phrase reveal functionality is not accessible via keyboard navigation.

### Steps to reproduce

Do this

### Error messages or log output

```shell

```

### Detection stage

On main branch

### Version

12.21.0

### Build type

None

### Browser

Brave

### Operating system

MacOS
